src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:3000/api" class="form form-big">
+      <div class="big">
```

Comment:
why do we need this additional wrapper?

Reply:
I use this block to insert image, so that I can use pseudo element "Before"